delta/plugins/elasticsearch/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/elasticsearch/config/IndexingConfig.scala

```diff
@@ -0,0 +1,11 @@
+package ch.epfl.bluebrain.nexus.delta.plugins.elasticsearch.config
+
+import ch.epfl.bluebrain.nexus.delta.kernel.RetryStrategyConfig
+
+/**
+  * Configuration for indexing process.
+  *
+  * @param concurrency indexing concurrency
+  * @param retry       indexing retry strategy configuration
+  */
+final case class IndexingConfig(concurrency: Int, retry: RetryStrategyConfig)
```

Comment:
Remove this. Is exists now in `ch.epfl.bluebrain.nexus.delta.kernel. IndexingConfig `